webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/NodeFactQueryProcessor.scala

```diff
@@ -99,7 +100,7 @@ object GroupAnd extends Group {
   }
   def inverse(a: NodeFactMatcher):                     NodeFactMatcher =
     NodeFactMatcher(s"!(${a.debugString})", (n: CoreNodeFact) => a.matches(n).map(!_))
-  val zero:                                            NodeFactMatcher = NodeFactMatcher("true", _ => true.succeed)
+  val zero:                                            NodeFactMatcher = NodeFactMatcher("and0_true", _ => true.succeed)
```

Comment:
changing the debug string just to make it clearer that that `true` (or the corresponding `false` in `orGroup` case is really the zero case of the group, not a constant coming from a matcher.